### PR TITLE
XW-1740 | remove '…is powered by…' sentence from footer

### PR DESCRIPTION
**Pull request by Warkot**
https://wikia-inc.atlassian.net/browse/XW-1740

@Wikia/x-wing

### CODE REVIEW

**Comment by kvas-damian** [comment]
Except failing unit tests it LGTM ;)

**Comment by rogatty** [comment]
LGTM

File: includes/wikia/models/tests/DesignSystemGlobalFooterModelTest.php [hunk 1]
```diff
@@ -97,39 +75,10 @@ public function licensingAndVerticalProvider() {
 			[
 				'CC-BY-SA',
```
Comment (Grunny) [inline includes/wikia/models/tests/DesignSystemGlobalFooterModelTest.php]: This test case is now the same as the first so can be removed.

**Comment by rogatty** [comment]
LGTM but let's run API tests before merging it.

**Comment by Warkot** [comment]
API tests passed (twice, same for @rogatty and me 😉 ). Merging.

```
warkot@mac-kot:~/wikia/api-tests (master)$ python main_runner.py --group="ds" --env="sandbox-so"
running on sandbox-so
...
----------------------------------------------------------------------
Ran 3 tests in 8.251s

OK
```

**Pull request merged**